src/app/app-routing.module.ts

```diff
@@ -31,7 +31,7 @@ const routes: Routes = [
   { path: 'user', loadChildren: './pages/menu/users/user-details/user-details.module#UserDetailsPageModule' },
   { path: 'user/:id', loadChildren: './pages/menu/users/user-details/user-details.module#UserDetailsPageModule' },
 
-  { path: 'home', loadChildren: './pages/home/home.module#HomePageModule' },
+  { path: '', loadChildren: './pages/home/home.module#HomePageModule' },
```

Comment:
Okay... definitely don't do this. For the same reason as before. We already have a blank path at line 5-9

Reply:
Okay...well I'll change it back, but after that I'm not sure what to do.